src/hooks/useConcentratorVaultTvl.ts

```diff
@@ -0,0 +1,14 @@
+import { ConcentratorVaultProps } from "@/lib/types"
+import useConcentratorVaultTvlFallback from "@/hooks/lib/tvl/concentrator/useConcentratorVaultTvlFallback"
+
+export function useConcentratorVaultTvl({
+  primaryAsset,
+  targetAsset,
+  type,
+}: ConcentratorVaultProps) {
+  return useConcentratorVaultTvlFallback({
```

Comment:
I feel like you don't need both of these hooks. You should just put the implementation from `useConcentratorVaultTvlFallback` in here.